src/connector/src/source/kafka/mod.rs

```diff
@@ -28,19 +28,19 @@ pub const KAFKA_CONNECTOR: &str = "kafka";
 
 #[derive(Clone, Debug, Deserialize)]
 pub struct KafkaProperties {
-    #[serde(rename = "kafka.brokers")]
+    #[serde(rename = "properties.bootstrap.server", alias = "kafka.brokers")]
```

Comment:
I feel 'brokers' is easier to comprehend

Reply:
Also Materialize uses the term 'brokers',

Reply:
In fact, they are different things, but FlinkSQL only supports `bootstrap.server`.

I'm not sure which one is better, but I don't want to review every name and I decide to follow all FlinkSQL terms :)

Reply:
https://stackoverflow.com/questions/50312579/what-is-difference-between-broker-list-and-bootstrap-servers